test/ha/workload_test.go

```diff
@@ -85,7 +85,7 @@ func TestActivatorInRequestPathPossibly(t *testing.T) {
 		rtesting.WithConfigAnnotations(map[string]string{
 			autoscaling.MinScaleAnnotationKey:  strconv.Itoa(minimumNumberOfReplicas), // Make sure we don't scale to zero during the test.
 			autoscaling.MaxScaleAnnotationKey:  strconv.Itoa(maximumNumberOfReplicas),
-			autoscaling.TargetBurstCapacityKey: "1", // The Activator may be in the path, depending on the revision scale and load.
+			autoscaling.TargetBurstCapacityKey: "200", // The Activator may be in the path, depending on the revision scale and load.
```

Comment:
default isn't actually 200 - it's 211

https://github.com/knative/serving/blob/10b9152229fed05b3e6676c9493d370d8818fc8a/config/core/configmaps/autoscaler.yaml#L100

If you remove the annotation you'll get teh default so you don't need to set it